prow/config.yaml

```diff
@@ -398,16 +398,7 @@ branch-protection:
                 # Require PR branch to be up-to-date with main
                 strict: true
         reconciler:
-          protect: false
-          branches:
-            main:
-              protect: true
-            develop:
-              protect: true
-            load-test-mock:
-              protect: true
-            istio-op:
-              protect: true
```

Comment:
Same here: I would only remove the branch protection if the reconciler repo is archived.